.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+require( '@automattic/eslint-plugin-wpvip/init' );
+
+module.exports = {
+	extends: [
+		'plugin:@automattic/wpvip/recommended',
+		'plugin:@automattic/wpvip/cli',
+		'plugin:@automattic/wpvip/weak-testing',
+		'plugin:@automattic/wpvip/weak-typescript',
```

Comment:
Does `plugin:@automattic/wpvip/typescript` generate a lot of errors?

Reply:
Does not seem like it. Resolved in ed1e6f8.